[DOCS-3379] Modify instructions for attaching permissions to integrat… (#13780)
DataDog/documentation: content/en/logs/log_configuration/archives.md

{{< tabs >}}
{{% tab "AWS S3" %}}

> [Create a policy][1] with the following two permission statements, and attach it to the role for the AWS Integration. Edit the bucket names and, if desired, specify the paths that contain your log archives.

Reviewer: ```suggestion
1. [Create a policy][1] with the following two permission statements:
```

> **Notes**:

Reviewer: ```suggestion
```

> * The `GetObject` and `ListBucket` permissions allow for [rehydrating from archives][2].
> * The `PutObject` permission is sufficient for uploading archives.

Reviewer: ```suggestion
   * The `GetObject` and `ListBucket` permissions allow for [rehydrating from archives][2].
   * The `PutObject` permission is sufficient for uploading archives.
```
... to make this a sublist of the suggested ordered list

}
```

> #### Attach the policy to the Datadog integration role

Reviewer: ```suggestion
```

> Attach the new policy to the Datadog integration role.

Reviewer: ```suggestion
2. Attach the new policy to the Datadog integration role.
```

> 1. Navigate to **Roles** in the AWS IAM console.
> 2. Locate the role used by the Datadog integration. By default it is named **DatadogIntegrationRole**, but the name may vary if your organization has renamed it. Click the role name to open the role summary page.
> 3. Click **Add permissions**, and then **Attach policies**.
> 4. Enter the name of the policy created above.
> 5. Click **Attach policies**.

Reviewer: ```suggestion
   a. Navigate to **Roles** in the AWS IAM console.
   b. Locate the role used by the Datadog integration. By default it is named **DatadogIntegrationRole**, but the name may vary if your organization has renamed it. Click the role name to open the role summary page.
   c. Click **Add permissions**, and then **Attach policies**.
   d. Enter the name of the policy created above.
   e. Click **Attach policies**.

3. Edit the bucket names.
4. Optionally, specify the paths that contain your log archives.
```

Author: @kayayarai thank you for this feedback. Within this context, it doesn't make sense to edit the bucket names or specify the log archive paths after the policy is attached, so I adjusted the order of these steps.
